conf/messages

```diff
@@ -292,41 +290,22 @@ business-reliefs.non-deductible.question.input.error.max-amount.agent = The amou
-error.supp.agent.unauthorised.link = Go back to account home
-error.unauthorised-user.heading = You are not authorised to use this service
-error.unauthorised.1 = You need to
-error.unauthorised.2 = create an agent services account
-error.unauthorised.2a = sign up for Making Tax Digital for Income Tax
-error.unauthorised.3 = before you can view this page.
-error.unauthorised.title = You cannot view this page
-internal-server-error-template.heading = Sorry, there is a problem with the service
-internal-server-error-template.paragraph.1 = Try again later.
-not-found-template.heading = Page not found
-not-found-template.paragraph.1 = If you typed the web address, check it is correct.
-not-found-template.paragraph.2 = If you used ‘copy and paste’ to enter the web address, check you copied the full address.
-service-unavailable-error-template.heading = Sorry, the service is unavailable
-service-unavailable-error-template.paragraph.1 = You will be able to use this service later.
-taxYear-error-template.paragraph.1 = You can only enter information for the tax years {0} to {1}.
-taxYear-error-template.paragraph.1.single = You can only enter information for a valid tax year.
-taxYear-error-template.paragraph.2 = Check that you’ve entered the correct web address.
-
-#################       Business Tax Reliefs messages        ########################################
+business-reliefs.qualifying-loan.title = Qualifying loan interest payable in the year
+business-reliefs.qualifying-loan.paragraph1.individual = You can claim tax relief for interest on a loan or alternative finance arrangement used to buy:
+business-reliefs.qualifying-loan.paragraph1.agent = Your client can claim tax relief for interest on a loan or alternative finance arrangement used to buy:
+business-reliefs.qualifying-loan.paragraph2.individual = The total amount of certain Income Tax reliefs that can be used to reduce your total taxable income is limited to £50,000, or 25% of your adjusted total income, whichever amount is greater.
+business-reliefs.qualifying-loan.paragraph2.agent = The total amount of certain Income Tax reliefs that can be used to reduce your client''s total taxable income is limited to £50,000, or 25% of your client''s adjusted total income, whichever amount is greater.
+business-reliefs.qualifying-loan.bullet1 = shares in, or to fund, a close company
+business-reliefs.qualifying-loan.bullet2 = an interest, or to fund, a partnership (if the partnership is a property letting partnership, follow the link below for further guidance)
+business-reliefs.qualifying-loan.bullet3.individual = plant or machinery for your work (unless you have already deducted this as a business expense)
+business-reliefs.qualifying-loan.bullet3.agent = plant or machinery for your client''s work (unless you have already deducted this as a business expense)
+business-reliefs.qualifying-loan.more-info-link = More information about interest and alternative finance payments eligible for relief
```

Comment:
From Hanah's comments we also need to update the content with a (opens in new page) added to the end of this string